### DISCUSSION TITLE
Nova License showing unregistered quite frequently

### DISCUSSION THREAD

**Discussion by RealityGamin** [laravel/nova-issues#4529] [Q&A]
I keep experiencing the unregistered nova logo, almost daily, sometimes no matter how many times I do `nova:check-license` it will not go away. This has been happening a lot in Nova 4, does anyone have any idea how to improve this? I honestly think it has something to do with my (CPanel & ConfigServer Security & Firewall (CSF)) settings, as they are very strict, or maybe not, can't tell honestly.

It is honestly extremely rare, that I go a full 24hrs with a fully licensed nova.

**Comment by murdercode** [comment]
Have you tried to launch ```php artisan optimize:clear```? It worked for me

**Comment by hitanshi2996** [comment]
I am facing the same,
even in localhost
